easybuild/easyconfigs/o/OpenMM/OpenMM-6.3-foss-2015a.eb

```diff
@@ -0,0 +1,26 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Author:    Stephane Thiell <sthiell@stanford.edu>
+##
+name = 'OpenMM'
+version = '6.3'
+
+homepage = 'http://openmm.org/'
+description = "A high performance toolkit for molecular simulation."
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+source_urls = ['https://github.com/pandegroup/openmm/releases']
+sources = [SOURCELOWER_TAR_GZ]
```

Comment:
hmm, this is not working for me...

should this be changed to:

```
source_urls = ['https://github.com/pandegroup/openmm/archive/']
sources = ['%(version)s.tar.gz']
```

https://github.com/pandegroup/openmm/archive/6.3.tar.gz works